web/app/js/components/Octopus.jsx

```diff
@@ -0,0 +1,99 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Col, Popover, Row } from 'antd';
+import { metricToFormatter, toShortResourceName } from './util/Utils.js';
+import './../../css/octopus.css';
+
+const displayName = resource => `${toShortResourceName(resource.type)}/${resource.name}`;
+
+const getDotClassification = sr => {
+  if (sr < 0.9) {
+    return "status-dot-poor";
+  } else if (sr < 0.95) {
+    return "status-dot-ok";
+  } else {return "status-dot-good";}
+};
+
+const Neighbor = ({neighbor, direction}) => {
+  return (
+    <div className="neighbor">
+      <Popover
+        title={displayName(neighbor)}
+        content={<MetricSummaryRow resource={neighbor} metricClass="metric-sm" />}
+        placement={direction ==="in" ? "left" : "right"}>
+        <div className="neighbor-row">
+          <div>{direction === "in" ? "<" : ">"}</div>
+          <div className={`status-dot ${getDotClassification(neighbor.successRate)}`} />
+          <div>{neighbor.name}</div>
```

Comment:
Nice! Thanks for adding the resource types. How would you feel about also using `displayName` here on line 28? Something like:

```
          <div>{displayName(neighbor)}</div>
```

That way the text next to the status bubble would match the title of the popover. As written it looks like this:

![screen shot 2018-08-21 at 2 29 36 pm](https://user-images.githubusercontent.com/9226/44430132-a7707f80-a54e-11e8-8196-afbcaf78c127.png)

Reply:
oh yeah, good catch!